scripts/auto-add-github-resources.py

```diff
@@ -0,0 +1,139 @@
+import sys
+import os
+from _github_utilities import create_branch, get_file_in_repository, get_issue_body, write_file, send_pull_request
+import yaml
+from github import Github
+
+def main():
+    """
+    Main function to retrieve metadata from a GitHub repository and append it to a YAML file. 
+    Creates a pull request with the updated data.
+
+    Arguments:
+    - repository (str): GitHub repository name.
+    - issue (int): Issue number containing the GitHub link.
+
+    Example:
+    python auto_add_github_entry.py "username/repo" 123
```

Comment:
```suggestion
    python auto_add_github_entry.py username/repo 123
```
I presume this is correct?

Reply:
I have deleted it. I wanted to test it locally or on my github, but it seems I'm not allowed to.